kafka-test/src/test/java/com/hazelcast/jet/tests/kafka/KafkaTest.java

```diff
@@ -178,6 +128,27 @@ public void tearDown() {
         jet.shutdown();
     }
 
+    private Pipeline pipeline() {
+        Pipeline pipeline = Pipeline.create();
+
+        pipeline.drawFrom(KafkaSources.<String, Trade>kafka(kafkaProps, topic))
+                .map(Map.Entry::getValue)
+                .addTimestamps(Trade::getTime, lagMs)
+                .window(sliding(windowSize, slideBy))
+                .groupingKey(Trade::getTicker)
+                .aggregate(counting())
+                .map(e -> {
```

Comment:
you can use the overload of `aggregate` here and skip this mapping step

Reply:
👍